build.gradle

```diff
@@ -0,0 +1,93 @@
+import com.github.spotbugs.SpotBugsTask
+
+plugins {
+  id "com.github.hierynomus.license" version "0.14.0" apply false
+  id 'com.github.johnrengelman.shadow' version '2.0.1' apply false
+  id "com.google.protobuf" version "0.8.3" apply false
+  id "com.github.spotbugs" version "1.6.0" apply false
+}
+
+allprojects {
+  apply plugin: 'idea'
+  group = 'cz.seznam.euphoria'
+  version = '0.8-SNAPSHOT'
+}
+
+subprojects {
+
+  apply plugin: 'java'
+  apply plugin: 'com.github.hierynomus.license'
+  apply plugin: 'com.github.spotbugs'
+  apply from: "${rootProject.projectDir}/gradle/versions.gradle"
+
+  license {
+    include '**/*.java'
+    header = rootProject.file('HEADER')
+    strictCheck = true
+    ext.year = Calendar.getInstance().get(Calendar.YEAR)
+    ext.name = 'Seznam.cz, a.s.'
+  }
+
+  repositories {
+    mavenCentral()
+  }
+
+  dependencies {
+    compileOnly "org.projectlombok:lombok:${lombokVersion}"
```

Comment:
Lombok should definitely not be top level dependency. Euphoria itself deliberately chose not to use it at all (as I see, it is as a dependency in `euphoria-hadoop`, it should be removed). I see it also in root pom, but it is not used in any code.

Reply:
ok, ill remove it for now, but we should reconsider ;)